src/components/picker/PickerItemsList.tsx

```diff
@@ -161,12 +162,26 @@ const PickerItemsList = (props: PickerItemsListProps) => {
     );
   };
 
+  const renderCustomTopElement = () => {
+    const {isMultiMode, value, setValue} = context;
+    if (customTopElement) {
+      if (isMultiMode) {
+        console.log(`renderCustomTopElement, isMultiMode!, value:`, value);
+        //@ts-expect-error - PickerWithMultiValue props need to pass the correct props
+        return customTopElement({value, setValue});
```

Comment:
We should avoid exposing internal workings on the components.
What if we want to change it in the future?

Reply:
You are right, fixed.